git_machete/cmd.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 
```

Comment:
I'd rename this entire file to sth like `main.py`... can be on a separate PR tho (since changes in other places where `cmd` is referenced will probably be needed as well)